src/guides/wiki/oba-2x-build-run-guide.md

```diff
@@ -16,12 +16,12 @@ OneBusAway Application modules are a suite of modules that assist in providing C
 
 v2.1 is intended for java11 on a Unix/Linux or mac platform.  As preconditions these instruction assume you have installed:
 * a recent git version
-* mvn 3.5.4
-* java 1.11.x
-* mysql 5.7
+* Maven 3.9.x
+* Java 11 (1.11.x) or later
+* MySQL 8.0 or compatible alternative (e.g., PostgreSQL)
 
 
-In a bash terminal:
+Steps to Build
```

Comment:
Did you run through these steps? There's a typo in the existing version that would prevent it from working properly. The `git checkout` line needs to be changed:

```
git clone https://github.com/OneBusAway/onebusaway-application-modules.git
cd onebusaway-application-modules
```

Reply:
> Now, how can I verify that everything is done correctly

I recommend following the tutorial and seeing if it works.